Sources/LiveViewNative/_GeneratedModifiers.swift

```diff
@@ -13562,6 +13562,7 @@ case let ._MatchedGeometryEffectModifier(modifier):
             
             let context: ParseableModifierContext
             
```

Comment:
This file seems auto-generated, so this fix isn't enough as it'd get reverted on the next generation.